spec/support/matchers.rb

```diff
@@ -236,6 +236,7 @@ def plugin_should_not_be_installed(*names)
     end
 
     def lockfile_should_be(expected)
+      expected = expected.dup.gsub!(%r{file:\/\/localhost}, "file://") if defined?(URI::File)
```

Comment:
why is this change necessary?

Reply:
`URI::File` normalize `localhost` params with https://github.com/ruby/ruby/blob/trunk/lib/uri/file.rb#L57

Reply:
Is that new in 2.6? If so, mind adding a comment ?